skrf/calibration/deembedding.py

```diff
@@ -1097,7 +1097,58 @@ def deembed(self, ntwk):
 
         return s_side1.inv ** ntwk ** s_side2.flipped().inv
 
-    def dc_interp(self, s, f):
+    def extrapolate_to_dc(ntwk):

```

Comment:
shouldn't it defined as @staticmethod ?

Reply:
same for other methods below where `self` has been suppressed?

Reply:
Indeed. They should probably need a docstring too.